src/components/FlatParameters/FlatParameters.js

```diff
@@ -211,6 +231,18 @@ export const FlatParameters = ({
         return value?.replace(',', '.') || '';
     };
 
+    const getStringListValue = (allValues, selectValues) => {
+        if (!Array.isArray(selectValues) || selectValues?.length === 0) {
```

Comment:
can selectedValues be with type other than array if defined?
if not you can simply use
if (!selectValues || !selectValues.length)